images/win/scripts/Installers/Install-Vcpkg.ps1

```diff
@@ -3,21 +3,23 @@
 ##  Desc:  Install vcpkg
 ################################################################################
 
-Import-Module -Name ImageHelpers -Force
+Write-Host "Download the latest vcpkg version..."
+$vcpkgDownloadUrl = (Invoke-RestMethod "https://api.github.com/repos/microsoft/vcpkg/releases/latest").tarball_url
+$installDir = 'C:\vcpkg'
+New-Item $installDir -type directory
+$tempPath = Join-Path -Path "${env:Temp}" -ChildPath "vcpkg.tar.gz"
+Invoke-WebRequest $vcpkgDownloadUrl -OutFile $tempPath
```

Comment:
Maybe use `Start-DownloadWithRetry`?

Reply:
And for releases manifest too.

Reply:
@vmapetr , it looks strange, but `Start-DownloadWithRetry` helper return unexpected response for this link:
```
The remote server returned an error: (403) Forbidden
```

Reply:
Thank you for your point, but when `Start-DownloadWithRetry` is being used we are getting `Exception calling "DownloadFile" with "2" argument(s): "The remote server returned an error: (403) Forbidden."`.
![error](https://user-images.githubusercontent.com/61747324/89031756-ca5f5d00-d33b-11ea-85b8-595e95bd1acb.jpg)